docs/src/main/asciidoc/reactive-messaging.adoc

```diff
@@ -199,29 +199,32 @@ Sending and publishing messages use the Vert.x event bus:
 
 [source, java]
 ----
-package org.acme;
+package org.acme.vertx;
 
-import io.vertx.axle.core.eventbus.EventBus;
-import io.vertx.axle.core.eventbus.Message;
+import io.smallrye.mutiny.Uni;
+import io.vertx.mutiny.core.eventbus.EventBus;
+import io.vertx.mutiny.core.eventbus.Message;
+import org.jboss.resteasy.annotations.jaxrs.PathParam;
 
 import javax.inject.Inject;
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-import java.util.concurrent.CompletionStage;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
 
 @Path("/async")
 public class EventResource {
 
-  @Inject
-  EventBus bus;                                                        // <1>
+    @Inject
+    EventBus bus;
 
-  @GET
-  @Path("/{name}")
-  public CompletionStage<String> hello(String name) {
-      return bus.<String>request("greeting", name)                             // <2>
-              .thenApply(Message::body);
-  }
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    @Path("{name}")
+    public Uni<String> greeting(@PathParam String name) {
+        return bus.<String>request("greeting", name)
+                .onItem().apply(Message::body);
+    }
 }
 ----
 <1> Inject the Event bus
```

Comment:
You removed the callouts in the code so the doc is failing because the 2 callouts are not found.

Reply:
Fixed